stdlib/universe/holt_winters/neldermead.go

```diff
@@ -237,14 +246,15 @@ func (o *Optimizer) Optimize(
 	// find the index of the smallest value
 	vs := 0
 	for i := 0; i <= n; i++ {
-		if f[i] < f[vs] {
+		if f.Value(i) < f.Value(vs) {
 			vs = i
 		}
 	}
 
-	parameters := make([]float64, n)
+	parameters := mutable.NewFloat64Array(o.alloc)
```

Comment:
Here it is, `Optimize` creates new params from the given ones. It cannot `Release` them for the very same reason of `forecast`, because otherwise they couldn't be used outside of its scope.